ReadMe.md

```diff
@@ -80,6 +80,30 @@ The following properties are currently supported.
 - `prefixRewrite` the URL path to modify the URL prefix path to
 - `targets` takes one target application for which the routes in a rule will be forwaded to. These can only be container app names.
 
+### Optional: Provisioning a Managed TLS/SSL Certificate for HTTPS
+
+If you are using a **custom domain** in your `httpRouteConfig` and wish to access it over **HTTPS**, you need to provision a **managed TLS/SSL certificate** in Azure Container Apps. If you have binded your custom domain to the path based routing spec using `auto` bindingType then once the TLS certificate is provisioned successfully, we will automatically update the certificate id field in the spec and bind it to your routing config within the managed environment.
```

Comment:
```suggestion
If you are using a **custom domain** in your `httpRouteConfig` and wish to access it over **HTTPS**, you need to provision a **managed TLS/SSL certificate** in Azure Container Apps. If you have bound your custom domain to the routing spec using `auto` bindingType then once the TLS certificate is provisioned successfully, it will automatically be added to the certificate id field in the spec and be bound to your routing config within the managed environment.
```